website/components/Footer.tsx

```diff
@@ -142,50 +142,36 @@ export function Footer() {
 }
 
 function SubmitForm() {
-  const [email, setEmail] = useState("");
   const router = useRouter();
 
-  const [message, setMessage] = useState("");
-
-  const subscribe = async (e: FormEvent) => {
+  const handleSubmit = (e: FormEvent) => {
     e.preventDefault();
-    // 3. Send a request to our API with the user's email address.
-    const res = await fetch(`/api/subscribe?email=${encodeURI(email)}`);
-
-    const { error } = await res.json();
-
-    if (error) {
-      // 4. If there was an error, update the message in state.
-      setMessage(error);
-
-      return;
-    }
-
     router.push("/confirm");
   };
 
   return (
     <form
```

Comment:
This is now an uncontrolled component, so no need to track the email input and errors